Cargo.toml

```diff
@@ -14,6 +14,12 @@ keywords = ["crc", "crc16", "crc32", "crc64", "hash"]
 categories = ["algorithms", "no-std"]
 edition = "2018"
 
+[features]
+default = ["slice16-defaults"]
```

Comment:
Can the `default = ["slice16-defaults"]` part be a separate PR? I think it would be good to have a release with the new functionality + feature flags before changing the defaults. I'm still not sure if this warrants a major version bump.

Reply:
Yes you are right, this should be able to be released as a minor version bump. So if no explicit action is taken users should still get the `bytewise` behavior.